experimental/packages/api-logs/src/types/LogRecord.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Attributes } from '@opentelemetry/api';
+
+export interface LogRecord {
+   /**
+    * The time when the log record occurred as UNIX Epoch time in nanoseconds.
+    */
+   timestamp?: number;
+
+   /**
+    * Numerical value of the severity.
+    */
+   severityNumber?: number;
+
+   /**
+    * The severity text.
+    */
+   severityText?: string;
+
+   /**
+    * A value containing the body of the log record.
+    */
+   body?: string;
+
+   /**
+    * Attributes that define the log record.
+    */
+   attributes?: Attributes;
+
+   /**
+    * 8 least significant bits are the trace flags as defined in W3C Trace Context specification.
+    */
+   traceFlags?: number;
+
+   /**
+    * A unique identifier for a trace.
+    */
+   traceId?: string;
+
+   /**
+    * A unique identifier for a span within a trace.
+    */
+   spanId?: string;
```

Comment:
What happens if I provide span context properties and already have `includeTraceContext` set to `true`

Reply:
Good question, I would expect that the provided context overrides what the instrumentation would otherwise add automatically. @scheler should this be added to the spec?

Reply:
My understanding is the span context should be added only by the sdk automatically. If it is possible you should prevent the end user from setting the span context properties directly. I noticed that the [Span](https://github.com/open-telemetry/opentelemetry-js-api/blob/main/src/trace/span.ts) interface does not allow setting them too.  In Java SDK, the [LogRecordBuilder](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk/logs/src/main/java/io/opentelemetry/sdk/logs/LogRecordBuilder.java) doesn't allow setting the span context properties.

Reply:
After a conversation with @scheler, I have removed the `includeTraceContext` option for now, as we could not think of a use case for it. If capable, the SDK should attach the active span context to any log record automatically, similar to how parent span context is attached to new spans. If the span context is added manually, it will take precedence over the active span context.